doc/api/stream.md

```diff
@@ -1886,8 +1886,8 @@ methods only.
 
 The `writable._writev()` method may be implemented in addition or alternatively
 to `writable._write()` in stream implementations that are capable of processing
-multiple chunks of data at once. If implemented, the method will be called with
-all chunks of data currently buffered in the write queue.
+multiple chunks of data at once. If implemented and if there are buffered data
```

Comment:
Nit: "are buffered data" is correct (because the word _data_ is plural) but treating _data_ as a singular noun has become idiomatic, is generally considered correct too, and seems to be what we do in the docs.  Treating _data_ as a singular word is so common that "data are" sounds jarring to many native speakers.

I am unable to find any other examples of "data" + "are" in our docs, but many "data" + "is":

buffer.md: "the data is zero-filled"
http2.md: "incoming data is not being read"
net.md: "when the data is finally" and "when data is received"
stream.md: "Writable streams are an abstraction for a *destination* to which data is..." and "Readable streams are an abstraction for a *source* from which data is..."
vm.md: "data is produced successfully"

...and many other examples.

Therefore, for consistency with the rest of the docs, I recommend switching this to "is":

```suggestion
multiple chunks of data at once. If implemented and if there is buffered data
```

Reply:
@Trott, done.